### PR TITLE
Subsystem sendable

### PR DESCRIPTION
**Pull request by vwarr**
Subsystems now have sendables 😁

### CODE REVIEW

File: src/main/java/frc/robot/constants/ArmConstants.java [hunk 1]
```diff
@@ -58,5 +58,23 @@ public enum ArmPositions {
...
+                    return "CONE_FLIP";
+                case SHOOT:
+                    return "SHOOT";
+                case PORTAL:
+                    return "PORTAL";
+                case HANDOFF:
+                    return "HANDOFF";
+            }
+            return "";
+        }
```
Comment (vwu1888) [inline src/main/java/frc/robot/constants/ArmConstants.java:78]: You don't need to override the built in toString of the enum. You can use enum.toString() or enum.name().

File: src/main/java/frc/robot/constants/CarriageConstants.java [hunk 2]
```diff
@@ -48,5 +48,14 @@ public enum CarriagePositions {
...
         }
+        public String toString() {
+            switch (this) {
+                case DOWN:
+                    return "DOWN";
+                case FLIPPED:
+                    return "FLIPPED";
+            }
+            return "";
+        }
```
Comment (vwu1888) [inline src/main/java/frc/robot/constants/CarriageConstants.java:59]: Same thing as the comment on the ArmConstants

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -108,6 +106,10 @@ public void robotPeriodic() {
         // and running subsystem periodic() methods.  This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
         CommandScheduler.getInstance().run();
+        SmartDashboard.putData(robotContainer.drivetrainSubsystem);
+        SmartDashboard.putData(robotContainer.armSubsystem);
+        SmartDashboard.putData(robotContainer.carriageSubsystem);
+        SmartDashboard.putData(robotContainer.elevatorSubsystem);
```
Comment (vwu1888) [inline src/main/java/frc/robot/Robot.java:112]: I'm pretty sure this shouldn't be in the robotPeriodic. I think that putData() publishes the entry into the table and should only be called once. If this is the case, let's put it in robotInit()

**Review by vwu1888 — DISMISSED** [review]

**Review by tsar-boomba — DISMISSED** [review]
LGTM 👍

**Review by tsar-boomba — APPROVED** [review]
LGTM 🔧 👍 👬 👭 💕 👍

**Pull request merged**